tests/test_select_implementation.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Nit: perhaps this test could go under `tests/python/relay/strategy/arm_cpu/`?

Reply:
Since the test checks for cpu and arm_cpu selections, I thought it was best to keep it under relay strategy so that the same test can be reused for multiple targets. Also, in future, the same can be expanded to cover other ops too. wdyt?

Reply:
Makes sense :)